### PR TITLE
Add recipe for sphinxbase

### PR DESCRIPTION
**Pull request by CurtLH**
Xref #6226

### CODE REVIEW

File: recipes/sphinxbase/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "sphinxbase" %}
+{% set version = "0.8" %}
```
Comment (CurtLH) [inline recipes/sphinxbase/meta.yaml]: I chose the version number based on the [`NEWS`](https://github.com/cmusphinx/sphinxbase/blob/master/NEWS) file in the upstream repo.  I'm happy to change it if someone has a better idea.

**Comment by CurtLH** [comment]
This recipe is ready for review @conda-forge/staged-recipes

File: recipes/sphinxbase/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/sphinxbase/meta.yaml]: hmm isn't the actual version `5prealpha`?

File: recipes/sphinxbase/meta.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {% set name = "sphinxbase" %}
-{% set version = "0.8" %}
+{% set version = "5prealpha" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
```
Comment (CJ-Wright) [inline recipes/sphinxbase/meta.yaml]: I don't think `5prealpha` is a valid conda-forge version. Would it be possible to write some jinja2 to convert `5prealpha` to a valid version? maybe `0.5.0.prealpha`?

Comment (isuruf) [reply recipes/sphinxbase/meta.yaml]: Ubuntu uses `0.8+5prealpha`

Comment (CJ-Wright) [reply recipes/sphinxbase/meta.yaml]: ah ok, would that be a valid version for us?

Comment (CurtLH) [reply recipes/sphinxbase/meta.yaml]: I'm open to whatever you guys think is best in this situation.

Comment (scopatz) [reply recipes/sphinxbase/meta.yaml]: probably `0.8.5prealpha`

**Comment by CurtLH** [comment]
Have we reached a consensus on what the version of this package should be @conda-forge/staged-recipes?  This is a dependency for another package that I'm trying to build, so hoping to keep this moving forward.

**Comment by scopatz** [comment]
Yes, this is fine @CurtLH - you just need to ping the issue by commenting in the conversation whenever it is ready to be looked at again.

**Pull request merged**